README.md

```diff
@@ -13,6 +13,7 @@
     <a href="./LICENSE">
         <img src="https://img.shields.io/github/license/citi/scaler?label=license&colorA=0f1632&colorB=255be3">
     </a>
+    <img alt="PyPI - Version" src="https://img.shields.io/pypi/v/scaler?colorA=0f1632&colorB=255be3">
```

Comment:
Could we make it clickable so that it links to https://pypi.org/project/scaler/ ?